### PR TITLE
Eslint, husky, deploy 

### PR DESCRIPTION
**Pull request by kostyalucker**
https://build-osbhdovkgl.now.sh/

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable no-undef */
```
Comment (imevro) [inline src/index.js]: нет

**Review by imevro — CHANGES_REQUESTED** [review]

**Comment by kostyalucker** [comment]
https://build-vwawgqgagp.now.sh/complex

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb",
+  "plugins": ["react", "jsx-a11y", "import"],
+  "rules": {
+    "react/prop-types": 0,
+    "linebreak-style": 0,
+    "avoidEscape": 0
+  },
+  "env": {
+      "jest": true,
```
Comment (imevro) [inline .eslintrc.json]: поч 4 пробела!!!

**Review by imevro — CHANGES_REQUESTED** [review]

**Comment by kostyalucker** [comment]
https://build-rfigbzsovh.now.sh/

**Review by imevro — APPROVED** [review]

**Pull request merged**